src/js/binaryen.js-post.js

```diff
@@ -255,6 +261,34 @@
     var literal = _malloc(16); // a single literal in memory. the LLVM C ABI
                                // makes us pass pointers to this.
 
+    function makeAtomicOps(ofType) {
+      return {
+        'rmw': {
```

Comment:
this could perhaps be more concise, since the `return [..]` line is almost identical on all these except one. So we could switch on just the `add => Module['AtomicRMWAdd']` parts, what do you think?

Reply:
Wasn't sure if there might be additional atomic ops in the future, besides `rmw`. Other than that, we'd also have to repeat the `{ 'rmw': ... }` four times below then - or did you mean to remove the `rmw` subsection entirely?

Reply:
Not sure what you mean by repeating that section, but maybe I wasn't clear before. This is what I meant, just to remove the duplication in those rmws by changing makeAtomicOps to this: https://gist.github.com/kripken/406fa34ab78a608a3442aaa8b0e77a40

Reply:
I see, thanks. This could be done for other functions as well. Though, the updated commit now uses the usual format similar to the other functions. Suggesting to leave the refactoring for another PR.

Reply:
Sure.